rocky/tools/forms/ooi_form.py

```diff
@@ -49,6 +50,10 @@ def generate_form_fields(
             if name == "primary_key":
                 continue
 
+            if name == "user_id":
+                fields[name] = forms.CharField(widget=forms.HiddenInput(), initial=str(self.user_id))
```

Comment:
I dont think we should let the user change this data. It should always be pinned to the actual logged in user in Rocky.
It can be good to show this on the form as a disabled field, to clarify to the user that they will be recorded as the 'creator' or the OOI, but if so we should probably do the same for the indemnification declarations.

Reply:
That is currently the case, the `user_id` is a hidden integer which needs to be added to the form (so that it can be added to the OOI). The user field is an unalterable value that display the username; it has no backend (see also image attached above).

Reply:
To add, I had the exact same discussion with @ammar92.

Reply:
I stand corrected: there was a way, addressed in 4333aa91a3d277404cc086f7cbdd6cfd77f9591d